PHP/PHP Source.sublime-syntax

```diff
@@ -373,29 +373,54 @@ contexts:
         1: keyword.other.use.php
       push:
         - meta_scope: meta.use.php
-        - match: (?=;|(?:^\s*$))
+        - match: (?=;|(?:\s*$)|\s+)
           pop: true
```

Comment:
This doesn't seem right. Does PHP really end a use statement as soon as a one or more spaces is present?